docker/images/pinot/Dockerfile

```diff
@@ -17,7 +17,7 @@
 # under the License.
 #
 
-FROM openjdk:8 AS pinot_build_env
```

Comment:
Try to add ARG JAVA_VERSION and in docker-build.sh pass JAVA_VERSION into the Dockerfile.